### PR TITLE
Modal profile menu

### PR DESCRIPTION
**Pull request by yepzdk**
https://platform.dandigbib.org/issues/5398

Add user profile menu in modal with seperate components for avatar, modal header and the profile modal itself.

Screenshot 2022-07-26 at 07 06 00

### CODE REVIEW

File: src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    },
+  },
+} as ComponentMeta<typeof ModalProfile>;
+
+const Template: ComponentStory<typeof ModalProfile> = (args) => (
+  <ModalProfile {...args} />
+);
+
+export const Item = Template.bind({});
+Item.args = {
```
Comment (Adamik10) [inline src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx:24]: Ever considered using argTypes with a default value set? (line 10) Then it's also added as a storybook control.
E.g. in the SearchResultItem.stories.tsx
```code
argTypes: {
    heartFill: {
      name: "Heart Fill", // name can also be added here for Storybook visualisation
      control: "boolean",
      defaultValue: false,
    },
```
(https://storybook.js.org/docs/react/api/argtypes)

File: src/stories/Library/avatar/Avatar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const StoryBase: StoryBaseType<typeof Avatar> = {
+  title: "Library / Avatar",
+  decorators: [withDesign],
+  argTypes: {},
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/xouARmJCONbzbZhpD8XpcM/Brugerprofil?node-id=1181%3A57001",
+    },
+    layout: "centered",
```
Comment (Adamik10) [inline src/stories/Library/avatar/Avatar.stories.tsx:14]: Nice! I didn't even know this existed. Good stuff 💯

**Review by Adamik10 — APPROVED** [review]
Very nice work :) Left a comment or two you can look at, but I'm approving 😉

File: src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx (hunk 1 above)
Comment (yepzdk) [reply src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx]: Thanks. That is absolutely the right way to do it. I have updated the story.

**Pull request merged**